ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -128,5 +118,17 @@ public void WhenInitialized_FindCellStateOfCell_UsingSourceData(CellState state,
 
             Assert.Equal(expected, result);
         }
+
+        [Fact]
+        public void testName() // test being used to test method 
+        {
+            var initData = "XXX" + Environment.NewLine + "XXX" + Environment.NewLine + "XXX";

```

Comment:
```suggestion
            var initData = "XXX" + Environment.NewLine +
                                   "XXX" + Environment.NewLine +
                                   "XXX";
```

you may find it easier to interpret like this